xml/Sector.xml

```diff
@@ -1562,7 +1802,7 @@
 				<narrative>Business support services and institutions</narrative>
 			</name>
 			<description>
-				<narrative>Support to trade and business associations, chambers of commerce; legal and regulatory reform aimed at improving business and investment climate; private sector institution capacity building and advice; trade information; public-private sector networking including trade fairs; e‑commerce. Where sector cannot be specified: general support to private sector enterprises (in particular, use code 32130 for enterprises in the industrial sector).</narrative>
+				<narrative>Support to trade and business associations, chambers of commerce; legal and regulatory reform aimed at improving business and investment climate; private sector institution capacity building and advice; trade information; public-private sector networking including trade fairs; e-commerce. Where sector cannot be specified: general support to private sector enterprises (in particular, use code 32130 for enterprises in the industrial sector).</narrative>
```

Comment:
it’s ‘e‑commerce’ vs ‘e-commerce’. The first uses a non-breaking hyphen. The new one is probably preferable.